android/codepush.gradle

```
@@ -60,8 +60,8 @@ gradle.projectsEvaluated {
             jsBundleDir = reactBundleTask.property('jsBundleDir').asFile.get()
             resourcesDir = reactBundleTask.property('resourcesDir').asFile.get()
             
-            new File(jsBundleDir).mkdirs()
-            new File(resourcesDir).mkdirs()
+            new File(jsBundleDir.toString()).mkdirs()
+            new File(resourcesDir.toString()).mkdirs()
```

```suggestion
            jsBundleDir.mkdirs()
            resourcesDir.mkdirs()
```
Cause jsBundleDir and resourcesDir are already instances of File